examples/run_adapt.py

```diff
@@ -96,6 +97,12 @@
     if "metric" not in out:
         break
     adj_sol, dwr, metric = out["adjoint"], out["dwr"], out["metric"]
+    
+    fig, axes = plt.subplots(1,2)
+    tricontourf(fwd_sol, axes=axes[0])
+    tricontourf(adj_sol, axes=axes[1])  
+    plt.savefig("out.jpg")
```

Comment:
I highly recommend visualisation in Paraview, rather than with matplotlib. If you really want to use matplotlib, I recommend using a more descriptive filename than `"out.jpg"`.